### ISSUE TITLE
How to allow dependencies to be found in multiple registries?

### ISSUE THREAD

**Issue by BenJWard** [JuliaComputing/Registrator.jl#64]
I've seen from this PR, https://github.com/JuliaComputing/Registrator.jl/pull/56, that multiple registries can be used for dependency resolution. Namely that the `register` function accepts a `registry_deps::Vector{String}` argument, however the `handle_register` function in the Server, currently does not call `register` with a `registry_deps` argument:

```julia
rbrn = register(pp.cloneurl, Pkg.Types.read_project(copy(IOBuffer(pp.projectfile_contents))), pp.tree_sha;
            registry=target_registry["repo"],
            push=true,
            gitconfig=Dict("user.name"=>config["github"]["user"], "user.email"=>config["github"]["email"]))
```

So when I tested the server locally (for a BioJulia registry) with a package that has deps in General I saw: 

```
Debug: cleaning up possibly inconsistent registry
│   registry_path = /Users/bward/repos/regist/image/scripts/registries/ccbd2cc2-2954-11e9-1ccf-f3e7900901ca
│   err = Error in `[deps]`: Package 'BioSymbols' with UUID: 3c28c6f8-a34d-59c4-9654-267d177fcfa9 not found in registry or stdlib
```

If I understand the code, it should be simple enough to allow a list of registries in the conf.toml, and let the server use them with the `register` method. Does that sound sensible? I can try this and PR it if it sounds reasonable.

**Comment by BenJWard** [comment]
I made a PR implementing this: #67

**Comment by nkottary** [comment]
Closed by #67
